packages/runtime/datastore/src/dataStoreRuntime.ts

```diff
@@ -388,48 +393,24 @@ IFluidDataStoreChannel, IFluidDataStoreRuntime, IFluidHandleContext {
     }
 
     public attachGraph() {
-        if (this.graphAttachState !== AttachState.Detached) {
+        if (this.localAttachState === AttachState.Attached) {
             return;
         }
-        this.graphAttachState = AttachState.Attaching;
+        this.localAttachState = AttachState.Attached;
         if (this.boundhandles !== undefined) {
             this.boundhandles.forEach((handle) => {
                 handle.attachGraph();
             });
             this.boundhandles = undefined;
         }
 
-        // Flush the queue to set any pre-existing channels to local
-        this.localChannelContextQueue.forEach((channel) => {
-            // When we are attaching the data store we don't need to send attach for the registered services.
-            // This is because they will be captured as part of the Attach data store snapshot
-            channel.markAttached();
-        });
-
-        this.localChannelContextQueue.clear();
-        this.bindToContext();
-        this.graphAttachState = AttachState.Attached;
-    }
-
-    /**
-     * Binds this runtime to the container
-     * This includes the following:
-     * 1. Sending an Attach op that includes all existing state
-     * 2. Attaching the graph if the data store becomes attached.
-     */
-     public bindToContext() {
-        if (this.bindState !== BindState.NotBound) {
-            return;
-        }
-        this.bindState = BindState.Binding;
         this.dataStoreContext.bindToContext();
```

Comment:
Leaving compat questins aside, should bindToContext() be called attachGraph()? :)